src/components/event/EventFilter/Event-filter.js

```diff
@@ -35,14 +51,9 @@ class EventFilter extends Component {
       this.state.needInitializeValues
     ) {
       this.props.initialize({
-        keyWord: initialValues.keyWord,
-        dateFrom: initialValues.dateFrom,
-        dateTo: initialValues.dateTo,
-        categories: initialValues.categories,
-        statuses: initialValues.statuses,
-        radius: initialValues.radius,
+        ...initialValues,
         selectedPos:
-          initialValues.selectedPos != null
+          initialValues.selectedPos !== null
```

Comment:
Simplify to `initialValues.selectedPos ? ... : ...`

Reply:
++